pkg/arvo/sys/vane/clay.hoon

```diff
@@ -412,12 +425,7 @@
 ++  print-wove
   |=  =wove
   :-  for.wove
-  ?-  -.rove.wove
-    %sing  [%sing mood.rove.wove]
-    %next  [%next [mood aeon]:rove.wove]
-    %mult  [%mult [mool aeon]:rove.wove]
-    %many  [%many [track moat]:rove.wove]
-  ==
+  ~
```

Comment:
+print-wove is constantly useful for `~&`s because requests tend to be too big to print.  I never leave home without it.